learnrl/callbacks/__init__.py

```diff
@@ -1,10 +1,11 @@
 import time
 import sys
 import numpy as np
+from learnrl.playground import Playground
```

Comment:
Same here, is the import necessary ?

Reply:
This file doesn't have Playground class. That's why I imported. If I remove it then how can it recognize Playground class

Reply:
It is using Playground class link in line 8

Reply:
Ok !

Reply:
Yeah after checking, you don't need to import itself to refer to it in documentation ! You just have to use the full name (`~learnrl.playground.Playground`)